src/NServiceBus.Core/Sagas/InvokeSagaNotFoundBehavior.cs

```diff
@@ -21,7 +21,7 @@ public async Task Invoke(IIncomingLogicalMessageContext context, Func<IIncomingL
                 return;
             }
 
-            logger.InfoFormat("Could not find a started saga for '{0}' message type. Going to invoke SagaNotFoundHandlers.", context.Message.MessageType.FullName);
+            logger.InfoFormat("Could not find a started saga of type '{1}' for '{0}' message type. Going to invoke SagaNotFoundHandlers.", invocationResult.SagaMetadata.Name, context.Message.MessageType.FullName);
```

Comment:
After the individual "Couldn't find a Saga X" messages from the other behavior, this message would only happen at the end if NO sagas were found and serves only to say we're invoking SagaNotFoundHandlers. But I have one tiny change to it and one grammar thing that bugs me.
```suggestion
            logger.InfoFormat("Could not find any started sagas for message type '{0}'. Going to invoke SagaNotFoundHandlers.", context.Message.MessageType.FullName);
```

Reply:
There could still be the case where the programmer did the right thing and only had one saga **type** in the endpoint handling the message but they still want to log the type of the saga where no saga instances were found, say because they are aggregating logs from all endpoints?

Reply:
Couldn't they use the other log statement for that?

Reply:
I just pushed a new change

- Removing the property in SagaInvocationResult 
- Removing the dummy saga in the unit test
- Moved the logging to  SagaPersistenceBehavior from InvokeSagaNotFoundBehavior
   - It still includes the saga type in what is being logged, but reading more closely [this other issue ](https://github.com/Particular/NServiceBus/issues/5992) it says sagas (in plural), I am going to send another change that will log all the sagas involved. The history of all the internal comments about this topic agrees that the handler is only going to be called if there aren't any saga found and that the way to "solve" this is by educating the user rather than trying to change the current behavior, especially after [Szymon comment](https://github.com/Particular/DeveloperExperience/issues/95#issuecomment-255089576)

Reply:
Alright, I just pushed another change, how about this so that we can fulfill both issues [5885](https://github.com/Particular/NServiceBus/issues/5885) and [5992](https://github.com/Particular/NServiceBus/issues/5992). While handling a message for a given saga type if that saga was not already started it will log exactly that, but the logged message doesn't say anything about whether or not the SagaNotFoundhandler is going to be invoked. Then another message later is logged, only if the saga not found handler is going to be invoked including the 'Any' working David suggested.  See sample log output in the following image
![image](https://user-images.githubusercontent.com/545695/152401039-311ce867-ce2d-4405-9b3f-f1100c8eaac1.png)

Now, if there are multiple sagas for the same message but one is not found and one is found here is how the log looks, only a message for the saga that was found non started.

![image](https://user-images.githubusercontent.com/545695/152402038-d5e1ab4d-e0f9-4842-9f65-125c9762a0fd.png)